src/spaces/dci-p3.js

```diff
@@ -0,0 +1,31 @@
+import Color from "./srgb.js";
+
+/**
+ * The Digital Cinema Initiative's DCI P3 uses a 6300-K reference whitepoint, 
+ * unlike Apple's Display P3 which uses the more common D65
+ *
+ * @see https://doi.org/10.5594%2FSMPTE.EG432-1.2010 SMPTE EG 432-1 
+ * @see https://doi.org/10.5594%2FSMPTE.RP431-2.2011 SMPTE RP 431-2
+ */
+Color.defineSpace({
+	inherits: "srgb",
+	id: "dci-p3",
+	name: "DCI P3",
+	cssId: "dci-p3",
+	white: Color.whites.Theater,
+	// Gamma correction is the same as sRGB
+	// convert an array of dci-p3 values to CIE XYZ
+	// using Theater {0.3140, 0.3510} instead of D65 = {0.3127, 0.3290} (no chromatic adaptation)
+	// http://www.brucelindbloom.com/index.html?Eqn_RGB_XYZ_Matrix.html
+	// Functions are the same as sRGB, just with different matrices
+	toXYZ_M: [
```

Comment:
Are these calculated directly from primaries and whites, or copied from Lindbloom?